plugins/database/quota/src/org/apache/cloudstack/api/command/QuotaStatementCmd.java

```diff
@@ -120,8 +119,8 @@ public String getCommandName() {
     @Override
     public long getEntityOwnerId() {
         Long accountId = _accountService.getActiveAccountByName(accountName, domainId).getAccountId();
```

Comment:
@rhtyd this will never be null. The method `getAccountId` returns a primitive long. Therefore, this check that you added is not needed. That is why when you asked to preserve the call `_accountService.getActiveAccountByName(accountName, domainId).getAccountId();` I wrote a code that was logging the value. Otherwise it would be a  call that did not make sense.

If you take a look at the first commit and compare it with HEAD, you are going to see the following code in HEAD:

>   if (accountId == null) {
>             return CallContext.current().getCallingAccount().getId();
>        }
>   return Account.ACCOUNT_ID_SYSTEM;

`accountId` is never null, that is why I removed that block, and left only the ` return Account.ACCOUNT_ID_SYSTEM;` code. If you think that we have to return the value of `accountService.getActiveAccountByName(accountName, domainId).getAccountId();` we can simply return it as it is a primitive long and will never be `null`

Reply:
Sure, let's return long value of accountId.

Reply:
Great. I will change this then